spacewalk/setup/bin/spacewalk-setup

```diff
@@ -854,6 +854,10 @@ sub populate_initial_configs {
      db_host => $answers->{'db-host'},
      db_port => $answers->{'db-port'},
      db_ssl_enabled => $answers->{'db-ssl-enabled'},
+     external_db => $answers->{'external-db'},
```

Comment:
I think migration https://github.com/uyuni-project/uyuni/blob/master/susemanager/bin/mgr-setup#L657 will not work in this first iteration

Reply:
also checking db_host != "localhost" or "hostname -f"  is not sufficient if they do not have a full dns name or what if they us the ip address.  Since this gets passed in via the setup-env.sh file, we would need more code to test all of this. seems setting a variable is much easier given how tangled all of this code already is.

Reply:
I fixed this temporarily now, but someone may want to revisit this to also add an ip address check.